app/portainer/components/tls-fieldset/index.ts

```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+
+import { TLSFieldset } from '@/react/portainer/environments/wizard';
+import { withFormValidation } from '@/react-tools/withFormValidation';
+import { tlsConfigValidation } from '@/react/portainer/environments/wizard/EnvironmentsCreationView/WizardDocker/APITab';
+
+import { tlsFieldsetAngular } from './tlsFieldsetAngular';
+
+export const ngModule = angular
+  .module('portainer.app.components.tls-fieldset', [])
+  .component('tlsFieldset', tlsFieldsetAngular);
+
+export const tlsFieldsetModule = ngModule.name;
+
+withFormValidation(
+  ngModule,
+  TLSFieldset,
+  'tlsFieldsetReact',
```

Comment:
this is the name you'll be using in angularjs, so it's expected not to include `react`
```suggestion
  'tlsFieldset',
```

Reply:
fixed

Reply:
It seems not right. the `tlsFieldsetReact` is referenced by `tlsFieldsetAngular` with 
```
...
  template: `
<ng-form name="$ctrl.tlsFieldset">
  <tls-fieldset-react
    values="$ctrl.values"
    on-change="$ctrl.handleChange"
    errors="$ctrl.errors"
  </tls-fieldset-react>
</ng-form>`,
...
```
If I change it to 'tlsFieldset`, the browser will show multiple directive error